Assets/Scripts/StoreManager.cs

```diff
@@ -113,6 +123,19 @@ public void ChangeStockCount(int UIIndex, int storeIndex, int change)
 
         totalCost = CalculateTotalCost();
         totalCostText.text = $"Total: ${totalCost}";
+
+        if (totalCost > GameManager.Instance.Money || totalCost <= 0)
```

Comment:
For some reason, the red colour for the total doesn't reset when you have no item in cart and $0. This was fixed if I changed it from `totalCost <= 0` to `totalCost < 0`, but this would then make the button green if you weren't buying anything.